package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-newrelic-lambda-layers",
-  "version": "0.1.18",
+  "version": "0.2.18",
```

Comment:
This change doesn't break API backwards compatibility. So could just be `0.1.19`.